src/sentry/reprocessing2.py

```diff
@@ -239,11 +239,11 @@ def reprocess_event(project_id, event_id, start_time):
     )
 
 
-def get_original_group_id(event):
+def get_original_group_id(event: Event) -> None | list | Mapping[str, None] | Sequence:
     return get_path(event.data, "contexts", "reprocessing", "original_issue_id")
 
 
-def get_original_primary_hash(event):
+def get_original_primary_hash(event: Event) -> None | list | Mapping[str, None] | Sequence:
```

Comment:
these might be better as just `-> object` -- since I can't imagine the caller can possibly unpack these values safely without narrowing anyway (I think these are just reaching into a schemaless json dict?)

Reply:
i think this should be narrowed to `str` using assertions within the function itself. likewise `get_original_group_id` should return `int`

Reply:
yeah, fixed it. you're both right